### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by LappleApple**
A few small changes. Nice, thorough guidelines!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -55,37 +55,37 @@ Thank you! We'll try to get back to you as soon as possible.
...
 
-   These are the same as when compiling diesel. In general, it is a good idea
-   to have _all_ drivers installed so you can run all tests locally.
+   These are the same as when compiling Diesel. It's generally a good idea
+   to install _all_ drivers so you can run all tests locally.
 
-   *Shortcut:* On macOS, you don't need to install anything to work with SQLite
-   and for PostgreSQL you'll only need the server (`libpq` is installed by
-   default). So, to get started, `brew install postgresql mysql` and follow the
+   *Shortcut:* On MacOS, you don't need to install anything to work with SQLite.
```
Comment (killercup) [inline CONTRIBUTING.md]: Apple writes macOS with a lowercase 'm' :)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -55,37 +55,37 @@ Thank you! We'll try to get back to you as soon as possible.
...
 4. Create a `.env` file in this directory, and add the connection details for
    your databases.
 
-   See [.env.sample](.env.sample) for an example that should work with a trivial
+   See [.env.sample](.env.sample) for an example that works with a trivial
    local setup.
 
    *Note:* If you didn't specify the MySQL user to be one with elevated
-   permissions, you'll want to a command like ```mysql -c "GRANT ALL ON
+   permissions, you'll want to make a command like ```mysql -c "GRANT ALL ON
```
Comment (killercup) [inline CONTRIBUTING.md]: Oh, totally missed the verb here, thanks! I think it should be 'run', though.

File: CONTRIBUTING.md (hunk 1 above)
Comment (LappleApple) [reply CONTRIBUTING.md]: Oh wow, thanks for that catch!

File: CONTRIBUTING.md (hunk 2 above)
Comment (LappleApple) [reply CONTRIBUTING.md]: +1

**Comment by killercup** [comment]
Thanks for the quick adjustments!

**Pull request merged**